odl/trafos/backends/pywt_bindings.py

```diff
@@ -481,52 +715,100 @@ def pywt_multi_level_recon(coeff_list, recon_shape, wavelet, mode):
         ``Di`` the tuple of i-th level detail coefficient arrays. Each of
         the ``Di`` tuples must have length ``2 ** ndim - 1``, where ``ndim``
         is the number of dimensions of the input array.
-    recon_shape : sequence of ints
-        Shape of the reconstructed array. This information is required since
-        this shape is not uniquely determined by the length of the
-        coefficients and the wavelet.
-    wavelet :  string or ``pywt.Wavelet``
+    wavelet :  string or `pywt.Wavelet`
         Specification of the wavelet to be used in the transform.
-        Use `pywt.wavelist
-        <https://pywavelets.readthedocs.io/en/latest/ref/wavelets.html#\
-built-in-wavelets-wavelist>`_ to get a list of available wavelets.
+        Use `pywt.wavelist` to get a list of available wavelets.
     mode : string, optional
-        PyWavelets style signal extension mode. See `signal extension modes
-        <https://pywavelets.readthedocs.io/en/latest/ref/signal-extension-\
-modes.html>`_ for available options.
+        PyWavelets style signal extension mode. See `signal extension modes`_
+        for available options.
+    recon_shape : sequence of ints, optional
+        Shape of the array to be reconstructed. Without this parameter,
+        the reconstructed array always has even shape due to upsampling
+        by a factor of 2. To get reconstructions with odd shapes, this
+        parameter is required.
 
     Returns
     -------
     recon : `numpy.ndarray`
         Wavelet reconstruction from the given coefficients.
+
+    See Also
+    --------
+    pywt_single_level_recon : Single-level version of this function.
+    pywt_multi_level_decomp : Multi-level decomposition, i.e. the inverse
+        of this function.
+
+    Examples
+    --------
+    Reconstruct the original array from the decomposition in the example
+    in `pywt_multi_level_decomp`:
+
+    >>> # Original array was
+    >>> # [[1, 1, 0, 0],
+    ... #  [0, 0, 0, 1],
+    ... #  [1, 1, 1, 1],
+    ... #  [0, 1, 1, 0]]
+    >>>
+    >>> approx2 = [[2.25]]
+    >>> details2 = ([[0.25]], [[-0.75]], [[0.25]])
+    >>> details1 = ([[0, -0.5],
+    ...              [-0.5, 0.5]],
+    ...             [[1, -0.5],
+    ...              [0.5, 0.5]],
+    ...             [[0, 0.5],
+    ...              [0.5, -0.5]])
+    >>>
+    >>> coeffs = [approx2, details2, details1]
+    >>> print(pywt_multi_level_recon(coeffs, wavelet='haar', mode='zpd'))
+    [[  1.00000000e+00   1.00000000e+00   3.92523115e-17   0.00000000e+00]
```

Comment:
Yeah, I'm working on it. I could instead compare to the original with `np.allclose`.